packages/core/src/js/feedback/FeedbackForm.types.ts

```diff
@@ -33,6 +33,12 @@ export interface FeedbackGeneralConfiguration {
    */
   showName?: boolean;
 
+  /**
+   * Should the screen shots field be included?
+   * Screen shots cannot be marked as required
+   */
+  enableScreenshot?: boolean;
```

Comment:
i: The name could change, since `enableScreenshot` doesn't give the impression that it will impact on the button visibility.
I thought about a bit verbose name, but when you read the property name, you know exactly what it will do without reading the description.
```suggestion
  showAddScreenshotButton?: boolean;
```

Reply:
That's indeed a good suggestion @lucas-zimerman 👍 
I'm a bit torn between [aligning](https://github.com/getsentry/sentry-react-native/pull/4328#discussion_r1871764316) with [the JS api](https://github.com/getsentry/sentry-javascript/blob/develop/packages/core/src/types-hoist/feedback/config.ts#L49) and using a more clear name for this 🤔

Reply:
I also think the name is too generic, but let stick with what users already know from JS.